processor/ratelimitprocessor/gubernator.go

```diff
@@ -102,17 +115,35 @@ func (r *gubernatorRateLimiter) RateLimit(ctx context.Context, hits int) error {
 	})
 	if err != nil {
 		r.set.Logger.Error("error executing gubernator rate limit request", zap.Error(err))
+		attrs := []attribute.KeyValue{
+			telemetry.WithErrorReason(telemetry.BadRequest),
+			telemetry.WithDecision("rejected"),
+		}
+		attrs = attrsFromMetadata(ctx, r.cfg.MetadataKeys, attrs)
```

Comment:
nit: move this in to a common function.

Reply:
Done in [b478e1b](https://github.com/elastic/opentelemetry-collector-components/pull/562/commits/b478e1b668b2843d6c65f2f6a53742b8807a8c8a)